packages/admin/resources/lang/pl/login.php

```diff
@@ -29,10 +31,8 @@
     ],
 
     'messages' => [
-        'failed' => 'Błędny login lub hasło.',
```

Comment:
The previous values were identical to one of the most popular translation packages for Laravel [https://github.com/Laravel-Lang/lang/blob/main/locales/pl/auth.php#L15](auth.php#L15)